workbench/runtime.py

```diff
@@ -201,7 +201,7 @@ def get_usage_id_from_aside(self, aside_id):
         try:
             return self._aside_usages[aside_id][0]
         except KeyError:
-            raise NoSuchUsage(usage_id)
+            raise NoSuchUsage(aside_id)
```

Comment:
This seems like a larger change. I don't know if it's correct, but I wanted to flag it and ask someone else who might know the difference to review it.